vueapp/src/App.vue

```diff
@@ -231,7 +241,37 @@ export default {
 				this.disableEditMode();
 				this.setLocked(true);
 			}
+		},
+		duplicateToggleDarkMode(id) {
+			console.log(id);
+			if(this.darkMode &&  DarkReader.isEnabled()) DarkReader.enable({ brightness: 100,
+				contrast: 90,
+				sepia: 10 });
+			else DarkReader.disable();
+		},
+		toggleDarkMode(id) {
+			console.log(id);
+			if (window) {
+				const {
+					enable: enableDarkMode,
+					disable: disableDarkMode,
+				} = require('darkreader');
+				if (this.dark || (!this.dark && this.systemDark)) {
```

Comment:
You can simplify this if-branch condition to `if (this.dark || this.systemDark)`

Reply:
Noted.